src/util.h

```diff
@@ -133,6 +131,16 @@ inline bool IsSwitchChar(char c)
 #endif
 }
 
+class ArgsManager
+{
+protected:
+    CCriticalSection cs_args;
+    std::map<std::string, std::string> mapArgs;
+    std::map<std::string, std::vector<std::string> > mapMultiArgs;
```

Comment:
I was just using the same that was used before. We can move to multimap, what would be the gain?